src/environment_cache.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"time"
+)
+
+const (
+	cachePath = "/.omp.cache"
+)
+
+type cacheObject struct {
+	Value     string `json:"value"`
+	Timestamp int64  `json:"timestamp"`
+	TTL       int64  `json:"ttl"`
+}
+
+type fileCache struct {
+	cache *concurrentMap
+	home  string
+}
+
+func (fc *fileCache) init(home string) {
+	fc.cache = newConcurrentMap()
+	fc.home = home
+	content, err := ioutil.ReadFile(fc.home + cachePath)
+	if err != nil {
+		return
+	}
+
+	var list map[string]*cacheObject
+	if err := json.Unmarshal(content, &list); err != nil {
+		return
+	}
+
+	for key, co := range list {
+		fc.cache.set(key, co)
+	}
+}
+
+func (fc *fileCache) close() {
+	fc.set("hello", "world", 200)
+	if dump, err := json.Marshal(fc.cache.list()); err == nil {
```

Comment:
format the output for human readability

Reply:
Is it possible to format it for human readbility with the addition of the timestamp?

Reply:
On my side it was serialized and thus impossible to read for a "human". Maybe we can add an extra parameter to omp to clear or display the content of the cache?